src/gui/usbipd_gui.rs

```diff
@@ -171,11 +171,30 @@ impl UsbipdGui {
                     .clone()
                     .unwrap_or_else(|| "Unknown Device".to_string()),
             );
-
             if device.is_bound() {
-                let menu_item = self
+                let menu_item;
+
+                if description.len() > 16 {
```

Comment:
It seems that this should be more like ~32. As is, the has the potential to make the name longer while conveying less information.

Reply:
Why not just increase the number to 32? Is it more complicated than that?